### PR TITLE
Adds burglar profession and swag bag

### PR DESCRIPTION
**Pull request by chaosvolt**
An idea I had in the past, decided to PR it. Point cost and whether to give any skills might need to be messed with, as well as whether to bother with the added item versus handing the profession a duffel bag.
- Burglar profession, with a basic array of entry tools and a stereotypical bag for loot.
- Added burglar to a few of the scenarios.
- Added ability to re-purpose large canvas sacks. If it weren't for the potential oddity of storing loot in bags full of powder, I would simply give regular canvas bags armor data.

### CODE REVIEW

**Comment by chaosvolt** [comment]
Thank you. Also at least should add to the uses of the humble canvas sack once it's emptied. I would've added a small version, except the small canvas sack has an immensely tiny storage capacity.

As for the profession itself, dunno yet if its point cost would warrant any starting skills, or whether it's fine as-is.

**Comment by chaosvolt** [comment]
Hnng. I hunger for feedback. ;w;

**Comment by chaosvolt** [comment]
Still hoping for feedback from @Coolthulhu or @Rivet-the-Zombie on the point cost, possible skill/item changes, etc. I suspect this PR got overlooked by the last couple waves of PRs that were examined. -w-

**Comment by Rivet-the-Zombie** [comment]
Yeah, I missed this one. Sorry about that!

I'd say the point cost is a bit low, considering how much handy stuff this profession starts with. Maybe raise it by a point or two?

**Comment by chaosvolt** [comment]
Ah, it's fine. ^^

And might be good to raise it by a point, sure.

File: data/json/items/armor.json [hunk 1]
```diff
@@ -10464,6 +10464,28 @@
...
+        "weight" : 415,
+        "color" : "brown",
+        "covers" : ["ARMS", "HANDS"],
+        "to_hit" : -5,
+        "storage" : 60,
+        "symbol" : "[",
+        "description" : "A large canvas sack, re-purposed for makeshift storage.  Goes well with a striped shirt and a domino mask.",
+        "price" : 0,
+        "material" : "cotton",
+        "volume" : 1,
```
Comment (Coolthulhu) [inline data/json/items/armor.json]: Just 1?

**Comment by chaosvolt** [comment]
Let me see, I suspect I based the swag bag's volume off of the item used to make it. If not, then it likely was an oversight when I edited whatever I copied the initial item off of. @_@

**Pull request merged**